src/server/scripts/Northrend/zone_icecrown.cpp

```diff
@@ -1472,6 +1472,48 @@ class spell_onslaught_or_call_bone_gryphon : public SpellScriptLoader
     }
 };
 
+enum OnslaughtGryphon
+{
+    SPELL_DELIVER_GRYPHON            = 54420,
+    SPELL_ONSLAUGHT_GRYPHON          = 49641,
+
+    NPC_CAPTURED_ONSLAUGHT_GRYPHON   = 29415,
+
+    SEAT_PLAYER                      = 0
+};
+
+class spell_deliver_gryphon : public SpellScript
+{
+    PrepareSpellScript(spell_deliver_gryphon);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DELIVER_GRYPHON, SPELL_ONSLAUGHT_GRYPHON });
```

Comment:
SPELL_DELIVER_GRYPHON is never used in the script.

Validate VEHICLE_SPELL_PARACHUTE)

Reply:
Also rename to SPELL_*

Reply:
this is out of context and will be removed.